templates/action-group.json

```diff
@@ -8,14 +8,25 @@
     "alertShortName": {
       "type": "string"
     },
-    "alertEmailReceivers": {
-      "type": "array"
+    "emailReceiverName": {
+      "type": "string",
+      "defaultValue": "Bulk print alerts",
```

Comment:
no I meant remove this, it shouldn't be cmc specific but that's an example of what you might set it to in the consuming code